wsgidav/dir_browser/_dir_browser.py

```diff
@@ -45,7 +45,11 @@ class WsgiDavDirBrowser(BaseMiddleware):
 
     def __init__(self, wsgidav_app, next_app, config):
         super(WsgiDavDirBrowser, self).__init__(wsgidav_app, next_app, config)
-        self.htdocs_path = os.path.join(os.path.dirname(__file__), "htdocs")
+        self.dir_config = config.get("dir_browser", {})
+        self.htdocs_path = os.path.realpath(self.dir_config.get('htdocs_path', os.path.join(os.path.dirname(__file__), "htdocs")))
+
+        if not os.path.exists(self.htdocs_path):
```

Comment:
`.isdir(...)` would be even more specific

Reply:
Changed!